bobtimus/src/coin_selection.rs

```diff
@@ -0,0 +1,259 @@
+use anyhow::Result;
+use bdk::{
+    bitcoin::Amount,
+    database::{BatchOperations, Database},
+    wallet::coin_selection::CoinSelectionResult,
+    wallet::coin_selection::{BranchAndBoundCoinSelection, CoinSelectionAlgorithm},
+};
+use elements_fun::{ExplicitTxOut, OutPoint};
+
+#[derive(Clone, PartialEq)]
+pub struct Utxo {
+    outpoint: OutPoint,
+    txout: ExplicitTxOut,
```

Comment:
This will need to be public so we can actually instantiate it outside of this module.